guides/common/modules/proc_exporting-a-content-view-version-incrementally-in-a-syncable-format.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Exporting_a_Content_View_Version_Incrementally_in_a_Syncable_Format_{context}"]
+= Exporting a Content View Version Incrementally in a Syncable Format
+
+Exporting complete Content View versions can be a very costly operation in terms of system resources.
+ifdef::orcharhino[]
+The size of the exported Content View versions depends on the number of products.
+endif::[]
+Content View versions that have multiple {RHEL} trees can occupy several gigabytes of space on {ProjectServer}.
+
+In such cases, you can use *Incremental Export* to export only pieces of content that changed since the previous export.
```

Comment:
```suggestion
In such cases, you can use *Incremental Export* to export only pieces of content that got changed since the previous export.
```

Reply:
I think it's probably best to leave as is so it doesn't sound past tense with `got`.